x/evm/keeper/migrations.go

```diff
@@ -11,3 +16,15 @@ func NewMigrator(keeper Keeper) Migrator {
 		keeper: keeper,
 	}
 }
+
+// Migrate1to2 migrates the store from consensus version v3 to v4
+func (m Migrator) Migrate1to2(ctx sdk.Context) error {
+	paramstore := m.keeper.paramSpace
+	if !paramstore.HasKeyTable() {
+		paramstore = paramstore.WithKeyTable(types.ParamKeyTable())
+	}
+
+	// add RejectUnprotected
+	paramstore.Set(ctx, types.ParamStoreKeyRejectUnprotected, types.DefaultParams().RejectUnprotected)
+	return nil
```

Comment:
let's move this logic to a `migrations/` dir (same as the `x/feemarket`)

Reply:
done